src/filters/frame_skipping_pp.py

```diff
@@ -0,0 +1,101 @@
+from src.filters.abstract_pp import AbstractPP
+from src.utils import image_utils
+from typing import List
+from enum import Enum, unique
+
+
+@unique
+class DistanceMetric(Enum):
```

Comment:
I guess a better place for DistanceMetric Enum is image_utils.

Reply:
Added it in image_utils as suggested